pkg/config/static/static_config.go

```diff
@@ -0,0 +1,90 @@
+package static
+
+import (
+	httpProvider "github.com/containous/maesh/pkg/proxy/provider/http"
+	traefikStatic "github.com/containous/traefik/v2/pkg/config/static"
+	"github.com/containous/traefik/v2/pkg/ping"
+	"github.com/containous/traefik/v2/pkg/provider/file"
+	"github.com/containous/traefik/v2/pkg/provider/rest"
+	"github.com/containous/traefik/v2/pkg/types"
+)
+
+// DefaultInternalEntryPointName the name of the default internal entry point
+const DefaultInternalEntryPointName = "traefik"
+
+// Configuration is the static configuration
```

Comment:
```suggestion
// Configuration is the Maesh proxy's static configuration
```

Could make sense to try to differentiate it to the traefik static configuration, wdyt?